algorithm.py

```diff
@@ -20,11 +24,9 @@
 
 Tutorial
 --------
-
-This module provides an abstraction for implementing arbitrary algorithms as a
-list of functions that operate on a shared state dictionary. Algorithms defined
-this way are easy to arbitrarily modify at run time, and they provide cascading
-exception handling.
+The algorithm in this module is a list of functions that are
+executed in defined order, the return values of which update the
+algorithm's run state in a `state` dictionary.
```

Comment:
Why didn't you use what I suggested at https://github.com/gratipay/algorithm.py/pull/12#discussion_r34317456?

> The model presented in this module is a list of functions that are executed in defined order, the return values of which update the algorithm's run state in a state dictionary.